Add FAQ section, change small errors (#5)
PicnicSupermarket/hiring-experience: src/pages/faq.adoc

== Frequently Asked Questions

After reading through, you couldn't find the answer to that one small question you have left? Or maybe you just want to know as much as possible about our process? Worry not! Below, you'll find all sorts of questions that we've been asked in the past (with an answer of course 😉).

=== How do I apply for a job?

> Follow the link to our https://join.picnic.app/jobs/[job page] and have a look at our openings. If you find any of them interesting, you can apply at the bottom of every ad. Make sure to give as much detail about yourself as you can, answer the questions (if your job as some) and click `Apply`. Simple!

Reviewer: "if your job as some" -> What do you mean here? :thinking:

[.text-right]
=== How long does the process take?

> The answer is dependent on a lot of factor, like your availability, how quickly you submit your assignment and where you live. On average, we are able to go from application to offer in a month. Not to brag though, but our record is 6 days. 🏆

Reviewer: ```suggestion
The answer is dependent on a lot of factors, like your availability, how quickly you submit your assignment and where you live. On average, we are able to go from application to offer in a month. Not to brag though, but our record is 6 days. 🏆
```

> The answer is dependent on a lot of factor, like your availability, how quickly you submit your assignment and where you live. On average, we are able to go from application to offer in a month. Not to brag though, but our record is 6 days. 🏆

Reviewer 2: ```suggestion
The answer is dependent on a lot of factors, like your availability, how quickly you submit your assignment and where you live. On average, we are able to go from application to offer in a month. Not to brag though, but our record is 6 days. 🏆
```

[.text-left]
=== What should I wear?

Whatever you feel like. We don't have a dresscode (whether for interviews or on a daily basis) so make sure you wear what you like. Our only rule is that we all should wear something...

[.text-right]
=== I need to travel to Amsterdam for the on-site vist? Are the costs covered?

> Yes! We take care of bookking your flights and hotel room and cover those costs. As part of the visit, you will most likely have lunch with us, however, meals (along with anythins else) taking place outside the office are on you.

Reviewer: ```suggestion
Yes! We take care of booking your flights and hotel room and cover those costs. As part of the visit you will most likely have lunch with us. However, meals (along with anything else) taking place outside the office are on you.
```

> Yes! We take care of bookking your flights and hotel room and cover those costs. As part of the visit, you will most likely have lunch with us, however, meals (along with anythins else) taking place outside the office are on you.

Reviewer 2: ```suggestion
Yes! We take care of booking your flights and hotel room and cover those costs. As part of the visit, you will most likely have lunch with us, however, meals (along with anything else) taking place outside the office are on you.
```

[.text-left]
=== What about the moving costs?

> When you move to Amsterdam to work at Picnic, we cover the costs of your flights (for you and your difrect family) and we put you up in one of the Picnic apartments (or hotel room should there be no apartment available) for one month, to give you time to find your own place while getting onboarded. 

Reviewer: ```suggestion
When you move to Amsterdam to work at Picnic, we cover the costs of your flights (for you and your direct family) and we put you up in one of the Picnic apartments (or hotel room should there be no apartment available) for one month, to give you time to find your own place while getting onboarded. 
```
